src/test/test-locale-util.c

```diff
@@ -50,9 +50,42 @@ static void test_locale_is_valid(void) {
         assert_se(!locale_is_valid("\x01gar\x02 bage\x03"));
 }
 
+static void test_get_keymaps(void) {
+        _cleanup_strv_free_ char **kmaps = NULL;
+        char **p;
+        int r;
+
+        r = get_keymaps(&kmaps);
+/*
+        assert_se(r >= 0);
+        assert_se(kmaps);
+
+        STRV_FOREACH(p, kmaps) {
+                puts(*p);
+                assert_se(keymap_is_valid(*p));
+        }
+*/
```

Comment:
instead of commenting this, I think we should just handle ENOENT cleanly here, i.e. exit early from this function